### DISCUSSION TITLE
Chlickhouse support.

### DISCUSSION THREAD

**Discussion by anasurimbo** [theory/sqitch#827] [Q&A] [answered]
Hello.
Is there any support for clickhouse?
Maybe, someone can share config for clickhouse connection?

Thanks!

**Comment by theory** [comment]
There is not. Would be great to add. Anyone got the TUITs for that?

**Comment by anasurimbo** [comment]
I know there is a trick to use postgres wire protocol, which allows us to use Postgres clients to connect to ClickHouse. But it would be great to add native connection.

**Comment by dchien234** [comment]
It would be nice if we can support this in `sqitch` as one of ClickHouse schema migration tools.
I looked at their options but nothing is as complete as `sqitch` https://clickhouse.com/docs/knowledgebase/schema_migration_tools.

**Comment by theory** [comment]
WIP: #902.

**Comment by theory** [comment, marked as answer]
Forgot to update here, but [v1.6.0](https://github.com/sqitchers/sqitch/releases/tag/v1.6.0) introduced official ClickHouse support.
